plugins/protocolbuffers/pyi/v27.0/buf.plugin.yaml

```diff
@@ -0,0 +1,18 @@
+version: v1
+name: buf.build/protocolbuffers/pyi
+plugin_version: v27.0
+source_url: https://github.com/protocolbuffers/protobuf
+description: Base types for Python Stubs. Generates stub files for message and enum types.
+output_languages:
+  - python
+spdx_license_id: BSD-3-Clause
+license_url: https://github.com/protocolbuffers/protobuf/blob/v27.0/LICENSE
+registry:
+  python:
+    package_type: "stub-only"
+    # https://github.com/protocolbuffers/protobuf/blob/v27.0/python/dist/setup.py#L78
+    requires_python: ">=3.8"
+    deps:
+      # https://pypi.org/project/protobuf/
+      - "protobuf~=5.27"
```

Comment:
Another manual step for now is keeping this updated.

Reply:
Maybe worth updating a readme or creating a known issue to remind ourselves. So when we come back to this we at least have something to reference (and also update as we add more automation over time).

Reply:
Added #1254 and #1255 to track the manual upgrades here.